### PR TITLE
[DEPS] Bumped adapters to 0.15.0 & SDK version to 0.26.0

### PR DESCRIPTION
**Pull request by harini-venkataraman**
## What

- Bumped adapters package to 0.15.0
- Bumped SDK version to 0.26.0

...

## Why

...

## How

...

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions / Env Variables

-

## Notes on Testing

...

## Screenshots

...

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: src/unstract/sdk/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = "0.25.1"
+__version__ = "0.25.2"
```
Comment (chandrasekharan-zipstack) [inline src/unstract/sdk/__init__.py:1]: @harini-venkataraman please assess if the changes add are breaking or not. Also FYI `0.25.1` itself has not been published yet.
Usually for minor version increments in adapters also we increment the minor version of SDK

Comment (gaya3-zipstack) [inline src/unstract/sdk/__init__.py:1]: SDK needs to go to 0.26.0 as we are supporting a new adapter.

Comment (harini-venkataraman) [reply src/unstract/sdk/__init__.py]: @chandrasekharan-zipstack Got it. Ignored the patch version and bumped the minor version itself.

**Review by gaya3-zipstack — APPROVED** [review]
Looks fine

**Review by chandrasekharan-zipstack — APPROVED** [review]

**Pull request merged**